arch/src/x86_64/layout.rs

```diff
@@ -69,9 +69,14 @@ pub const MEM_32BIT_RESERVED_SIZE: GuestUsize = (1024 << 20);
 
 // == Fixed constants within the "32-bit reserved" range ==
 
-// Sub range: 32-bit PCI devices (start: 3GiB, length: 768Mib)
+// Sub range: 32-bit PCI devices (start: 3GiB, length: 640Mib)
 pub const MEM_32BIT_DEVICES_START: GuestAddress = MEM_32BIT_RESERVED_START;
-pub const MEM_32BIT_DEVICES_SIZE: GuestUsize = (768 << 20);
+pub const MEM_32BIT_DEVICES_SIZE: GuestUsize = (640 << 20);
+
+// PCI MMCONFIG space (start: after the device space, length: 256MiB)
```

Comment:
How this 256MiB was chosen? Is it standard?
Just a clarification, is this where we're going to memory map the extended config space of PCI devices with an actual extended space?

Reply:
256MiB is the standard PCI MMCONFIG space for one segment. It's based on multiplying the total potential BDFs (2**16) by the size per BDF which is 4096.

Reply:
Oh I see, thanks for the details.